docs/contributors-guide.md

```diff
@@ -82,6 +78,17 @@ We are more interested to see how you go about fixing the issue. We want to know
 
 Take your time to learn and follow the workflow to the letter. Do not skip any steps because you think that step is "not important". We are more impressed when you finish an issue in fewer attempts than when you finish it in a shorter time but take many attempts because you were not meticulous enough along the way.
 
+## Roles
```

Comment:
What about moving this to just before the explanation on the knowledge required? Then it would flow better.